luigi/contrib/bigquery.py

```diff
@@ -371,9 +391,11 @@ def copy(self,
 
 
 class BigQueryTarget(luigi.target.Target):
-    def __init__(self, project_id, dataset_id, table_id, client=None, location=None):
+    def __init__(self, project_id, dataset_id, table_id, client=None, location=None, enable_chunking=False, chunk_size_gb=1000):
```

Comment:
These should be `BigQueryLoadTask` parameters, not `BigQueryTarget` constructor's parameters.

The purpose of a `Target` is identifying and locating an output, and help in achieving some properties 
 like idempotency. Better not couple the aspects of the output's _production_ in it, as producing output is what a `Task` does.